ros/processors/report_processor.py

```diff
@@ -0,0 +1,110 @@
+import json
+import requests
+import logging
+import tarfile
+from io import BytesIO
+from http import HTTPStatus
+from confluent_kafka import Consumer
+from ros.lib.host_inventory_interface import fetch_host_from_inventory
+from ros.app import app, db
+from ros.config import INSIGHTS_KAFKA_ADDRESS
+from ros.models import PerformanceProfile
+
+LOG = logging.getLogger(__name__)
```

Comment:
I don't see anywhere in the processor file you ave used it.

Reply:
Oh yes, I was going to implement the logger in ROS instead of using the print statement but I think we should now implement logger in some other PR. For now, I will remove the logger.

Reply:
Sounds good. Thank you @patilsuraj767